### PR TITLE
recommend usage of trusted package sources

### PR DESCRIPTION
**Pull request by kartheekp-ms**
Updated docs to recommend usage of trusted package sources.

### CODE REVIEW

File: docs/concepts/Security-Best-Practices.md [hunk 1]
```diff
@@ -117,7 +117,7 @@ For more information about Dependabot alerts & security updates, [see the follow
 
 **📦 Package Consumer**
 
-When using multiple public & private NuGet source feeds, a package can be downloaded from any of the feeds. To ensure your build is predictable and secure from known attacks such as [Dependency Confusion](https://medium.com/@alex.birsan/dependency-confusion-4a5d60fec610), knowing what specific feed(s) your packages are coming from is a best practice. You can use a single feed or private feed with upstreaming capabilities for protection.
+Use package sources that you trust. When using multiple public & private NuGet source feeds, a package can be downloaded from any of the feeds. To ensure your build is predictable and secure from known attacks such as [Dependency Confusion](https://medium.com/@alex.birsan/dependency-confusion-4a5d60fec610), knowing what specific feed(s) your packages are coming from is a best practice. You can use a single feed or private feed with upstreaming capabilities for protection.
```
Comment (nkolev92) [inline docs/concepts/Security-Best-Practices.md:120]: at the team meeting we talked about ensuring that we've added this in all places where we have relevant documentation.

Comment (zivkan) [reply docs/concepts/Security-Best-Practices.md:120]: thanks, I was meaning to comment on the PR, but lost track of time and forgot.  Examples of other places where calling out sources should be trusted include the nuget.config reference docs, and nuget sources add, and dotnet nuget add source.  Maybe also other docs that discuss package sources, like the page on using sources that need authentication.

Comment (kartheekp-ms) [reply docs/concepts/Security-Best-Practices.md:120]: > nuget.config reference docs, and nuget sources add, and dotnet nuget add source

Thank you for the pointers. Pushed new changes. I will create a separate PR for dotnet nuget add/update source.

**Review by nkolev92 — APPROVED** [review]

**Review by zivkan — APPROVED** [review]

File: docs/reference/cli-reference/cli-ref-sources.md [hunk 2]
```diff
@@ -54,13 +57,16 @@ where `<operation>` is one of *List, Add, Remove, Enable, Disable,* or *Update*,
...
   > [!NOTE]
-  > Be aware that encrypted passwords are only supported on Windows. 
+  > Be aware that encrypted passwords are only supported on Windows.
   > Moreover, they can only be decrypted on the same machine and by the same user who originally encrypted them.
 
 - **`-src|-Source`**
 
   Path to the package(s) source.
 
+  > [!NOTE]
```
[the remark this answers is not in the document]
Comment (kartheekp-ms) [reply docs/reference/cli-reference/cli-ref-sources.md]: Make sense Nikolche. I removed the note at the option level.

**Pull request merged**